### DISCUSSION TITLE
Remove "MSE" Text from top right of camera stream when using live/webrtc/stream.html

### DISCUSSION THREAD

**Discussion by pacmac** [blakeblackshear/frigate#6656] [Ask A Question]
Have searched for this, but could not find anyone asking the same question.

When I stream from
```
http://myip:5000/live/webrtc/stream.html?src=door_low
```

In the top right corner I get the text: "MSE" and I cannot figure out how to remove it:

<img width="503" alt="Screenshot 2023-05-31 at 9 35 32 AM" src="https://github.com/blakeblackshear/frigate/assets/2349856/c0d56e51-41bc-4b40-a7d0-a047b02c70f8">

Does anybody know ??

**Comment by NickM-27** [comment]
This is embedded in go2rtc, it can't be removed.
